packages/engine/Source/Scene/MetadataComponentType.js

```diff
@@ -113,9 +114,7 @@ const MetadataComponentType = {
  */
 MetadataComponentType.getMinimum = function (type) {
   //>>includeStart('debug', pragmas.debug);
-  if (!MetadataComponentType.isNumericType(type)) {
-    throw new DeveloperError("type must be a numeric type");
-  }
+  Check.typeOf.string("type", type);
```

Comment:
The `@exception` tag should be removed from these functions.

Reply:
Thanks for catching that. Fixed.